.stylelintrc

```diff
@@ -1,9 +1,6 @@
 {
-  "extends": "stylelint-config-standard",
+  "plugins": ["stylelint-prettier"],
   "rules": {
-    "at-rule-no-unknown": null,
-    "font-family-no-missing-generic-family-keyword": null,
-    "no-descending-specificity": null,
-    "color-hex-length": "long"
```

Comment:
I'd like to keep this rule `"color-hex-length": "long"` as it will really prevent us from adding custom colors to the codebase in the future.

https://github.com/prettier/stylelint-prettier#options says code editors will ignore custom configs but as least `npm test` and Travis will still respect them.

Reply:
makes sense, although I'd also strongly advocate that we start replacing any "raw color value" in our CSS (outside of the color variable definitions) with named variables. In a different issue of course.